src/sdk/pynni/nni/compression/torch/activation_rank_filter_pruners.py

```diff
@@ -35,10 +35,13 @@ def __init__(self, model, config_list, optimizer=None, activation='relu', statis
 
         super().__init__(model, config_list, optimizer)
         self.set_wrappers_attribute("if_calculated", False)
-        self.set_wrappers_attribute("collected_activation", [])
+        self.set_wrappers_attribute("collected_activation", None)
```

Comment:
we should do deep copy within `set_wrappers_attribute` or add document to tell users if passing an object, the object will be shared by all the wrappers. @chicm-ms what do you think?

Reply:
I think deep copy is too much, maybe we can consider to remove `set_wrappers_attribute` if it is too tricky to use it. Both `if_calculated` and `collected_activation` can be implemented without `set_wrappers_attribute`.

Reply:
@chicm-ms It does not solve the problem, I think we should add docstring for API to make it clear